src/helper.js

```diff
@@ -454,6 +454,17 @@ export function titleCase(string) {
   return split.join(" ");
 }
 
+export function renderRaceTier(tierNumber) {
+  let res = "";
+  for (let i = 0; i < tierNumber; i++) {
+    res += "●";
+  }
+  for (let i = 0; i < 4 - tierNumber; i++) {
+    res += "○";
+  }
+  return res;
```

Comment:
This could be simplified with:

```js
return "●".repeat(tierNumber) + "○".repeat(Math.max(0, 4-tierNumber));
```

Reply:
lol I already did this

Reply:
wait what is the `Math.max(0` for?

Reply:
isn't is already positive?

Reply:
@nstringham in case in the future they will add a tier above 4... so it won't go `.repeat(-1)` and give error

Reply:
Ofc we would need to update the "4" of "4-tierNumber" to fix it... but in the meanwhile the profiles won't load

Reply:
`return "●".repeat(tierNumber) + "○".repeat(Math.max(0, 4-tierNumber));`

So I learnt thats a thing... didn't know about the repeat function until today. thx :)